Common/services/NewsService.swift

```diff
@@ -210,7 +211,7 @@ class NewsService: HTTPService {
 
         group.wait()
 
-        let task = self.request(method: .GET, path: "search/\(encodedText)", params: nil, handler: httpHandler)
+        let task = self.request(method: .GET, path: "search/\(encodedText)?page=\(page)", params: nil, handler: httpHandler)
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Canillitapp/headlines-iOS/blob/45d8ea94c6e792106316d76b5f17306d72f6aea0/Common/services/NewsService.swift#L214"></span>Line should be 120 characters or less: currently 124 characters<br />
<code>line_length</code> <code>NewsService.swift:214</code></td>
    </tr>
  </tbody>
</table>